### ISSUE TITLE
[Docs] Clicking the contributing breadcrumb from the Meshery Server page leads to a 404

### ISSUE THREAD

**Issue by hershd23** [meshery/meshery#4930]
#### Current State

Clicking the contributing breadcrumb (See top of first image) leads to a 404
![Screenshot 2022-01-12 at 12 23 10 AM](https://user-images.githubusercontent.com/37028207/149003903-cf68d651-30f8-4689-8d76-1fa75c41e929.png)
![Screenshot 2022-01-12 at 12 23 26 AM](https://user-images.githubusercontent.com/37028207/149003946-15627623-17fb-4db6-8a34-abba44219c99.png)

On link inspection you can see that the actually see that the error is due to a trailing /

Actual link : `https://docs.meshery.io/project/contributing`
#### Desired State

Error link: `https://docs.meshery.io/project/contributing/`

Simply changing to the correct link should fix this issue
---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by varunsaral** [comment]
Working on it , submitting a PR

**Comment by kulsoomzahra** [comment]
hey @leecalcote can i take this up?

**Comment by ashutosh887** [comment]
Hey @leecalcote  Should I try on this one?

**Comment by warunicorn19** [comment]
hey @ashutosh887, I'll have to assign this one to @kulsoomzahra since she commented first. Please feel free to take a look at other issues 😃

**Comment by ashutosh887** [comment]
No Issues @warunicorn19 
I can help if she wants

**Comment by varunsaral** [comment]
@warunicorn19 i was working on it sir? i have submitted 3 PR already but was failed , i was about to submit one today.

**Comment by ashutosh887** [comment]
Should I try it now? @warunicorn19

**Comment by Akshit42-hue** [comment]
Can I work on this issue @asubedy

**Comment by Akshit42-hue** [comment]
Can I work on this issue @asubedy

**Comment by asubedy** [comment]
Considering @ashutosh887 commented first, @ashutosh887 if you still want to work on this you can. 

If not @Akshit42-hue  can take this.

**Comment by ashutosh887** [comment]
I'll do it @asubedy

**Comment by Akshit42-hue** [comment]
Sure @ashutosh887 go ahead if you want any help feel free to ping me

**Comment by Akshit42-hue** [comment]
Any update @ashutosh887 ?

**Comment by Akshit42-hue** [comment]
Should I go ahead? @asubedy

**Comment by asubedy** [comment]
Let's give some time to @ashutosh887 to reply, if there is no reply in next 24 hours, you can go ahead @Akshit42-hue

**Comment by ashutosh887** [comment]
Working
Need some help!

**Comment by asubedy** [comment]
@ashutosh887 you can ask for help in slack channel we will be happy to help you, you can ask questions here too.

**Comment by asubedy** [comment]
Freeing assignment @Akshit42-hue please take this forward.

**Comment by Akshit42-hue** [comment]
The link is working fine please do have a look @asubedy

**Comment by asubedy** [comment]
@Akshit42-hue, yes It seems @leecalcote already solved the issue sneakily 😆 , alright then please look into some other issues, thank you. Closing the issue
